src/paint_canvas/cached_chunk.rs

```diff
@@ -0,0 +1,5 @@
+#[derive(Clone)]
+pub struct CachedChunk {
+   pub png: Vec<u8>,
+   pub webp: Option<Vec<u8>>,
+}
```

Comment:
I don't really see a reason for this to be in its own file, I think it better fits living next to `cache_layer::CacheLayer`.